src/theme/components/progressIndicator.ts

```diff
@@ -53,9 +53,13 @@ const ProgressIndicator = defineMultiStyleConfig({
         circularContainer: {
           alignItems: "center",
           display: "flex",
-          flexDirection: "column",
+          flexDirection: labelPlacement === "bottom" ? "column" : "row",
           width: "fit-content",
         },
+        circularLabel: {
```

Comment:
Maybe this should be `circularLabelRight`? Or something that indicates it is only styles for when the label is on the right hand side.